lib/OpenQA/WebAPI.pm

```diff
@@ -286,8 +286,8 @@ sub startup ($self) {
     $admin_r->get('/needles/:module_id/:needle_id')->name('admin_needle_module')->to('needle#module');
     $admin_r->get('/needles/ajax')->name('admin_needle_ajax')->to('needle#ajax');
     $admin_r->delete('/needles/delete')->name('admin_needle_delete')->to('needle#delete');
-    $admin_r->get('/auditlog')->name('audit_log')->to('audit_log#index');
-    $admin_r->get('/auditlog/ajax')->name('audit_ajax')->to('audit_log#ajax');
+    $op_r->get('/auditlog')->name('audit_log')->to('audit_log#index');
+    $op_r->get('/auditlog/ajax')->name('audit_ajax')->to('audit_log#ajax');
```

Comment:
I don't think this is a good idea. The audit log can reveal personal information and enable something like "tracking employee performance" hence it is on purpose limited to admins. We should not allow that for the big group of operators. We need to think of a better solution here.

Reply:
Ok, so I guess I'll implement a special "limited" audit log route where the user parameter is always the current user. That I could actually make accessible to all users so the audit log would no longer have to be operator-only.

Reply:
That sounds perfect!